Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Formats/Format.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats
+{
+    public enum Format
```

Comment:
There isn't a way to create one control to be "universal" for any text entry. What happens if I want this toolbar for a regular Textbox and not a RichTextbox? TextBox still supports bold, italics, hyperlinks, colors, etc. 

Having code paths like 
```
if(Format == RichText)
{
    DoRichText();
}
else if(Format == Markdown)
{
    DoMarkdown();
}
else if(Format == Textbox)
{
    DoTextBox()
}
```
Is a perfect example of violating the Open/Closed principle of software development. 

If I'm the only person who thinks it should be "more generic" then throw my suggestion away